src/javascript/app/base/header.js

```diff
@@ -157,12 +171,34 @@ const Header = (() => {
             el.src = Url.urlForStatic(`${header_icon_base_path}ic-close.svg`);
         });
 
+        applyToAllElements('#header__notification-icon', (el) => {
+            el.src = Url.urlForStatic(`${header_icon_base_path}ic-bell.svg`);
+        });
+
+        applyToAllElements('#header__notification-empty', (el) => {
```

Comment:
Should this selectors match to that from line 111? If yes, then it should be `header__notification-empty-img`
Not sure, if this comment is relevant, but I noticed that all the rest selectors have a pair from constants above

Reply:
yeah i was applying to container instead of image tag. i fixed it